docs/manual/tags.md

```diff
@@ -2,7 +2,38 @@
 layout: default
 title: rpm.org - RPM Tags
 ---
-# RPM Tag Listing
+# RPM Tags
+
+The package's meta data in stored in the RPM header. The header is a binary data structure that stores the single pieces of data in tags. Each tag has a pre-defined meaning and data type. These are not stored in the header itselfs but need to be known by the code reading the header. In the header the tags are only refered by their number.
+
+## Tag types
+
+### Scalar types
+
+There are four unsigned integer types for RPM tags:
+
+* `int8`
+* `int16`
+* `int32`
+* `int64`
+
+`int8` and `int16` are rarely used. Most integer tags are `int32` with a few `int64` replacements where 32 bits were just not enough. These being unsigned limits time stamps to 1970-01-01 and later.
```

Comment:
Leave this paragraph out, eg the timestamp mention is only confusing in this context, and whether a type is common or not isn't really relevant. I'd rather point out that these are technically always arrays.